lib/reporter/index.js

```diff
@@ -108,10 +108,19 @@ class Reporter extends SimplifiedReporter {
     this.testResults.reportPrefix = prefix;
   }
 
+  setSessionInfo(data) {
+    this.testResults.driverCapabilities = data.capabilities;
```

Comment:
@prudhvi22  are we sure, the selenium server does not return back any authentication information like username, password?

Reply:
Yes, It is not returning any sensitive information.

Reply:
okay